### PR TITLE
getDOMNode() has been deprecated. Use React.findDOMNode() instead

### PR DESCRIPTION
**Pull request by vasanthk**
https://facebook.github.io/react/blog/2015/03/10/react-v0.13.html
https://facebook.github.io/react/docs/component-api.html#getdomnode

Also, event.which is deprecated too. Use event.keyCode instead. (event.key is still not supported by few browsers including Chrome)

### CODE REVIEW

File: examples/react/js/app.jsx [hunk 1]
```diff
@@ -35,17 +35,17 @@ var app = app || {};
 		},
 
 		handleNewTodoKeyDown: function (event) {
-			if (event.which !== ENTER_KEY) {
+			if (event.keyCode !== ENTER_KEY) {
```
Comment (samccone) [inline examples/react/js/app.jsx]: why this change?

Comment (vasanthk) [inline examples/react/js/app.jsx]: @samccone Does it look good for merging?

File: examples/react/js/app.jsx [hunk 2]
```diff
@@ -35,17 +35,17 @@ var app = app || {};
...
 				return;
 			}
 
 			event.preventDefault();
 
-			var val = this.refs.newField.getDOMNode().value.trim();
+			var val = React.findDOMNode(this.refs.newField).value.trim();
```
Comment (passy) [inline examples/react/js/app.jsx]: The indentation here is off now. Could you fix that please?

Comment (vasanthk) [inline examples/react/js/app.jsx]: My bad.. Fixed indentation.

File: examples/react/js/app.jsx [hunk 3]
```diff
@@ -35,17 +35,17 @@ var app = app || {};
...
 
 			if (val) {
 				this.props.model.addTodo(val);
-				this.refs.newField.getDOMNode().value = '';
+        		React.findDOMNode(this.refs.newField).value = '';
```
Comment (arthurvr) [inline examples/react/js/app.jsx]: This indentation is wrong.

Comment (arthurvr) [inline examples/react/js/app.jsx]: We use tabs, the first 'tab' here are actually spaces.

**Comment by arthurvr** [comment]
It looks like this introduced some new failing specs. Can you please run the tests and check what's wrong?

**Comment by vasanthk** [comment]
@arthurvr Not really sure. Travis Ci indicates a chrome driver issue.. crashing
Any pointers?

**Comment by samccone** [comment]
@vasanthk you an ignore the integration test failures, we are waiting on a feature upstream in travis and sauce labs until they work

**Comment by passy** [comment]
@vasanthk This doesn't work yet for me.

screenshot 2015-06-12 09 24 00

**Comment by arthurvr** [comment]
I wasn't complaining about CI, just run the tests locally and you'll see the failing specs.

**Comment by vasanthk** [comment]
@passy It was since an older version of React was installed. Updated package.json to the latest. Should work fine now. Let me know if any issues.

**Comment by passy** [comment]
@vasanthk You need to also update the bundled dependencies, otherwise it's still 0.12 this is run against.

**Comment by vasanthk** [comment]
@passy Gotcha.. Updated it. Let me know if there is anything else. Thanks :)

**Comment by vasanthk** [comment]
@passy @arthurvr @samccone Squashed it all into a single commit. Let me know if it looks good for merge. Thanks :)

**Comment by arthurvr** [comment]
> Let me know if it looks good for merge.

Did you run the tests? This introduced a bunch of new failures.

**Comment by vasanthk** [comment]
@arthurvr Thanks bud. Figured out the issue and fixed it. Tests run good now. Let me know if the issue persists.

**Comment by arthurvr** [comment]
Neat work @vasanthk!
